website/content/en/preview/concepts/disruption.md

```diff
@@ -7,152 +7,122 @@ description: >
 ---
 
 ## Control Flow
+
 Karpenter sets a Kubernetes [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) on each node it provisions.
-The finalizer blocks deletion of the node object while the Termination Controller cordons and drains the node, before removing the underlying machine. Deprovisioning is triggered by the Deprovisioning Controller, by the user through manual deprovisioning, or through an external system that sends a delete request to the node object.
-
-### Deprovisioning Controller
-Karpenter automatically discovers deprovisionable nodes and spins up replacements when needed. Karpenter deprovisions nodes by executing one [automatic method](#methods) at a time, in order of Expiration, Drift, Emptiness, and then Consolidation. Each method varies slightly but they all follow the standard deprovisioning process:
-1. Identify a list of prioritized candidates for the deprovisioning method.
-   * If there are [pods that cannot be evicted](#pod-eviction) on the node, Karpenter will ignore the node and try deprovisioning it later.
-   * If there are no deprovisionable nodes, continue to the next deprovisioning method.
-2. For each deprovisionable node, execute a scheduling simulation with the pods on the node to find if any replacement nodes are needed.
+The finalizer blocks deletion of the node object while the Termination Controller cordons and drains the node, before removing the underlying NodeClaim. Disruption is triggered by the Disruption Controller, by the user through manual disruption, or through an external system that sends a delete request to the node object.
+
+### Disruption Controller
+
+Karpenter automatically discovers disruptable nodes and spins up replacements when needed. Karpenter disrupts nodes by executing one [automatic method](#methods) at a time, in order of Expiration, Drift, Emptiness, and then Consolidation. Each method varies slightly, but they all follow the standard disruption process:
```

Comment:
I think this anchor link needs to be updated now. Can you change it to the other type of anchorlinking present elsewhere in this doc?
```suggestion
Karpenter automatically discovers disruptable nodes and spins up replacements when needed. Karpenter disrupts nodes by executing one [automatic method](#automatic_methods) at a time, in order of Expiration, Drift, Emptiness, and then Consolidation. Each method varies slightly, but they all follow the standard disruption process:
```

Reply:
should also remove emptiness since it's not listed as a method anymore
```suggestion
Karpenter automatically discovers disruptable nodes and spins up replacements when needed. Karpenter disrupts nodes by executing one [automatic method](#methods) at a time, in order of Expiration, Drift, and then Consolidation. Each method varies slightly, but they all follow the standard disruption process:
```

Reply:
Nice catch!